### DISCUSSION TITLE
video tutorial

### DISCUSSION THREAD

**Discussion by pashameow** [AveYo/ShowMMR#4] [Ideas]
make it)

**Comment by AveYo** [comment]
Have you clicked [Releases](https://github.com/AveYo/ShowMMR/releases) link yet?  

Cause I've provided this:
[video with install steps on youtube, optional ShowMMR tool steps included](https://youtu.be/-7FoalzztFI)  

It's not a full tutorial, but should be enough to get anyone going, it's not rocket-science ;)
